README.md

```diff
@@ -249,6 +240,168 @@ class UserController extends Controller {
 }
 ```
 
+## If your API doesn't follow our conventions
+Now it's time to introduce you some annotations helping configuring own routes. These class annotations must only be used for entities. The following entity is the one created in the chapter before, but with the additional ```DataSource```annotations.
+
+```php
+namespace CircleBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Circle\DoctrineRestDriver\Annotations as DataSource;
+
+/**
+ * This annotation marks the class as managed entity:
+ *
+ * @ORM\Entity
+ * @ORM\Table("products")
+ * @DataSource\Select("http://www.yourSite.com/api/products/findOne/{id}")
+ * @DataSource\Fetch("http://www.yourSite.com/api/products/findAll")
+ * @DataSource\Insert("http://www.yourSite.com/api/products/insert")
+ * @DataSource\Update("http://www.yourSite.com/api/products/update/{id}")
+ * @DataSource\Delete("http://www.yourSite.com/api/products/remove/{id}")
+ */
+class Product {
+
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+    
+    /**
+     * @ORM\Column(type="string", length=100)
+     */
+    private $name;
+    
+    public function getId() {
+        return $this->id;
+    }
+    
+    public function setName($name) {
+        $this->name = $name;
+        return $this;
+    }
+    
+    public function getName() {
+        return $this->name;
+    }
+}
+```
+
+By using these annotations the driver won't send the requests to the default urls, but to the ones configured. You don't need to use all of these annotations if you just want to define a specific route for one method. The annotation will only affect the configured HTTP method. The ```{id}``` string is a placeholder for the entities identifier:
```

Comment:
The annotations tell the driver to send the requests to the configured URLs for each custom configuration. If you just want to ..., you don't need to use all annotations provided. The `{id}` act as a placeholder for the entity's identifier.